### ISSUE TITLE
The 25.9. update breakes the Tutorial

### ISSUE THREAD

**Issue by paplhjak** [Jingkang50/OpenOOD#188]
### TLDR:
CLIP installation is not optional and needs to be added to the tutorial notebook.

### Full Issue
After the update on 25 Sept 2023, the tutorial notebook can not be run.
> A unified, easy-to-use evaluator that allows evaluation by simply creating an evaluator instance and calling its functions. Check out this [colab tutorial](https://colab.research.google.com/drive/1tvTpCM1_ju82Yygu40fy7Lc0L1YrlkQF?usp=sharing)!

The code returns the following exception `ModuleNotFoundError: No module named 'clip'`. See the image below for details.
![image](https://github.com/Jingkang50/OpenOOD/assets/52961799/7d1e4e66-f25c-4e5d-a334-33df5e3e325a)

I suspect that CLIP was not added to the repository dependencies. Installation of CLIP is hence not optional, and the following installation instructions should be changed.
```
pip install git+https://github.com/Jingkang50/OpenOOD
# optional, if you want to use CLIP
# pip install git+https://github.com/openai/CLIP.git
```
If one wants to run the tutorial notebook, he will also need to add `pip install git+https://github.com/openai/CLIP.git` to the Google Collab.

**Comment by zjysteven** [comment]
Hi @paplhjak, thank you for reporting the bug! You are right. I've added a try-except fix at [989c41d](https://github.com/Jingkang50/OpenOOD/commit/989c41dd3e3ebe85e6f5a5702acb6ce555715735) which should make things work again. Please have another try and let us know if there are any other issues.
